### ISSUE TITLE
Flatpak references EOL Runtime

### ISSUE THREAD

**Issue by ShobuPrime** [flathub/com.synology.SynologyDrive#26]
When updating my Flatpaks with `flatpak update`, I receive the following notice:

```
Info: runtime org.freedesktop.Sdk branch 20.08 is end-of-life, with reason:
org.freedesktop.Platform 20.08 is no longer receiving fixes and security updates. Please update to a supported runtime version.
Info: applications using this runtime:
com.synology.SynologyDrive

Info: runtime org.freedesktop.Platform.GL.default branch 20.08 is end-of-life, with reason:
org.freedesktop.Platform 20.08 is no longer receiving fixes and security updates. Please update to a supported runtime version.
Info: applications using this extension:
com.synology.SynologyDrive
```

Application itself still runs fine on Arch with latest kernel, but updating would still be recommended.

**Comment by bonjour81** [comment]
Same issue on my side (Manjaro). In addition to the obsolesence/Security issue, it forces to install different version of freedesktop packages taking a lot of disk space.

**Comment by jibsaramnim** [comment]
This issue can be closed now, as it's resolved with #25.

**Comment by RayJW** [comment]
Has been solved for quite some time.
